oak_containers_orchestrator/src/container_runtime.rs

```diff
@@ -145,11 +151,21 @@ pub async fn run(container_bundle: &[u8]) -> Result<(), anyhow::Error> {
         cmd
     };
 
+    log::info!("CWD: {:?}", oci_filesystem_bundle_config.process.cwd);
+    log::info!("ENV: {:?}", oci_filesystem_bundle_config.process.env);
+
     let prep_trusted_app_process = move || {
-        // Run the trusted app in a chroot enviroment of the container.
+        // Run the trusted app in a chroot environment of the container.
         std::os::unix::fs::chroot(container_rootfs_path.clone())?;
-        // Set the specified working directory
-        std::env::set_current_dir(oci_filesystem_bundle_config.process.cwd.clone())?;
+        let cwd = oci_filesystem_bundle_config.process.cwd.clone();
+        if cwd.has_root() {
+            std::env::set_current_dir(cwd)?;
+        }
+        for variable in oci_filesystem_bundle_config.process.env.clone() {
+            if let Some((key, value)) = variable.split_once('=') {
```

Comment:
Do you assume the given enviroment variables to be exactly in this format: VAR=VAL ?

What about e.g., VAR = VAL or VAR= VAL? Will this be a problem? 

Is there some library support for parsing enviroment variables or can you add documentation?

I think I remember spending a couple on hours on tracking down that " FOO" is not "FOO".

Reply:
These come from the container image. If there was a space, these would be included when running the container in e.g. Docker as well, and I don't think we should change the behaviour.